src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__calendar calendar calendar--start-day_sun calendar--month-length_31">
```

Comment:
You should follow BEM naming convention block-name--modifier-name--modifier-value
```suggestion
    <div class="page__calendar calendar calendar--start-day-sun calendar--month-length-31">
```

Reply:
Fix comment